ArduCopter/mode_flowhold.cpp

```diff
@@ -136,7 +136,7 @@ void ModeFlowHold::flowhold_flow_to_angle(Vector2f &bf_angles, bool stick_input)
     // compensate for height, this converts to (approx) m/s
     sensor_flow *= constrain_float(height_estimate, height_min, height_max);
 
-    // rotate controller input to earth frame
```

Comment:
Txs for the renaming.

Is this comment an improvement though?  I think it's rotating the flow sensor's input from body-frame into earth-frame.  I guess it's just a difference between "rotate" and "convert".

Anyway, I'm actually not a fan of this mode in any case, it's quite sensor and frame specific and so it leads to a fair number of support calls on the forums.

Reply:
I do believe this is an improvement because "rotate to earth frame" can mean two opposite things depending on the context. The rewording is unambiguous and clear.

Now, for this particular comment in flowhold, it's pretty obvious what's going on, so this one comment arguably didn't need the rewrite.

But let's take an example of one of the other comments which I think is dangerously ambiguous:
```
// rotate a 2D vector from earth frame to body frame
Vector2f earth_to_body2D(const Vector2f &ef_vector) const;
```
If the body frame is rotated 90 degrees to the right, does this function rotate the input vector 90 degrees to the right? Or does it rotate it 90 degrees to the left? It's ambiguous. It depends on if the author means active rotation transformation or passive. In this case, if you carefully inspect what the function is they mean passive, so rotating 90 degrees to the left. It's way clearer to just say that you are converting an earth-frame representation to a body-frame representation; yes you apply a rotation transformation to achieve that conversion, but I think "rotate" is a very tricky word in this context.